### PR TITLE
Adding a human touch to external facing messages

### PR DESCRIPTION
**Pull request by taliafr0st**
Command modification suggestions resulting from https://discord.com/channels/93055209017729024/203232366456602624/1225034850688303216

n.b. Untested

### CODE REVIEW

File: ftsbot/cogs/textcommands.py [hunk 1]
```diff
@@ -427,21 +427,30 @@ async def on_liquipedia_error(
...
 			usewiki = interaction.channel.name
-		if usewiki is not None:
+		if usewiki is None:
+			await interaction.response.send_message(
+				embed=discord.Embed(
+					colour=discord.Colour(0xff0000),
+					description='No wiki specified'
+				)
+			)
+		if user is None:
```
Comment (hjpalpha) [inline ftsbot/cogs/textcommands.py:453]: imo `elif` here

Comment (FO-nTTaX) [reply ftsbot/cogs/textcommands.py:453]: Could also just be an `else`, no?

Also error messages should be ephemeral like 
```await interaction.response.send_message(embed=discord.Embed(...), ephemeral=True)```

File: ftsbot/cogs/textcommands.py [hunk 2]
```diff
@@ -547,17 +570,39 @@ async def on_thinking_error(
...
+				embed=discord.Embed(
+					colour=discord.Colour(0x00ffff),
+					description=(
+						'Please have the copyright owner email the image to "photos@liquipedia.net", '
+						+ 'alongside a statement giving their permission for it to be used on Liquipedia. '
+						+ 'Please also include what wiki the image is for and the player in question.'
+					)
+				)
+			)
 		await interaction.response.send_message(
```
Comment (FO-nTTaX) [inline ftsbot/cogs/textcommands.py:592]: The second block would need to be an `else:` or there would be two messages I think.

File: ftsbot/cogs/textcommands.py (hunk 1 above)
Comment (hjpalpha) [reply ftsbot/cogs/textcommands.py:453]: there is an else below

atm it is

if
if
else

i suggest
if
elif
else

so we do not get 2 messages in some cases

Comment (FO-nTTaX) [reply ftsbot/cogs/textcommands.py:453]: oh I thought you wanted 
```
if wiki in data.wikis:
			usewiki = wiki
		elif isinstance(interaction.channel, discord.channel.TextChannel) and interaction.channel.name in data.wikis:
			usewiki = interaction.channel.name
		else:
			await interaction.response.send_message(
				embed=discord.Embed(
					colour=discord.Colour(0xff0000),
					description='No wiki specified'
				)
			)
```

Comment (FO-nTTaX) [reply ftsbot/cogs/textcommands.py:453]: basically this would catch usewiki not being set right where it's set

Comment (FO-nTTaX) [reply ftsbot/cogs/textcommands.py:453]: so I wanted to roll it into the previous if/elif and you wanted to roll it into the later if/elif

Comment (hjpalpha) [reply ftsbot/cogs/textcommands.py:453]: my point is that
```
if usewiki is None:
	send message
if user is None:
	send message
else
	send message
```

will (try to) send 2 messages if usewiki is None
(and produce a bs link in the 2nd message)

while
```
if usewiki is None:
	send message
elif user is None:
	send message
else
	send message
```
doesn't have that issue

alternative of couse would be
```
if usewiki is None:
	send message
else
	if user is None:
		send message
	else
		send message
```

Comment (taliafr0st) [reply ftsbot/cogs/textcommands.py:454]: Resolved conditionals by performing early returns, lmk if that's not preferred

Comment (FO-nTTaX) [reply ftsbot/cogs/textcommands.py:455]: This is fine

**Pull request merged**